tests/python/gaia-ui-tests/gaiatest/runtests.py

```diff
@@ -49,10 +50,27 @@ class GaiaTextTestRunner(MarionetteTextTestRunner):
     resultclass = GaiaTestResult
 
 
+class GaiaMarionetteDriver(Marionette):
```

Comment:
If I understand this correctly, it changes the default sandbox from 'default' to 'system' within GaiaTestCase? This might cause confusion. Is there a reason we don't change all of our execute_*_script calls to include the desired sandbox?